src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -981,18 +984,15 @@ def _validate_vm_vmss_create_public_ip(cmd, namespace):
         namespace.public_ip_address_type = 'new'
         logger.debug('new public IP address will be created')
 
-    from azure.cli.core.profiles import ResourceType
-    PublicIPAddressSkuName, IPAllocationMethod = cmd.get_models('PublicIPAddressSkuName', 'IPAllocationMethod',
-                                                                resource_type=ResourceType.MGMT_NETWORK)
     # Use standard public IP address automatically when using zones.
     if hasattr(namespace, 'zone') and namespace.zone is not None:
-        namespace.public_ip_sku = PublicIPAddressSkuName.standard.value
+        namespace.public_ip_sku = 'Standard'
 
     # Public-IP SKU is only exposed for VM. VMSS has no such needs so far
     if getattr(namespace, 'public_ip_sku', None):
-        if namespace.public_ip_sku == PublicIPAddressSkuName.standard.value:
+        if namespace.public_ip_sku == 'Standard':
             if not namespace.public_ip_address_allocation:
-                namespace.public_ip_address_allocation = IPAllocationMethod.static.value
+                namespace.public_ip_address_allocation = 'Static'
```

Comment:
Just out of curiosity, may I ask whether the values defined as enumerations in Swagger can continue to be defined as enumerations in Code Gen?